lib/reports/subscriber_lists_report.rb

```diff
@@ -2,7 +2,7 @@ class Reports::SubscriberListsReport
   attr_reader :date, :slugs, :tags_pattern, :links_pattern, :headers
 
   def initialize(date, slugs: "", tags_pattern: nil, links_pattern: nil, headers: nil)
-    @date = Time.zone.parse(date)
+    @date = Time.zone.parse(date)&.to_date
```

Comment:
Could this be `Date.parse(date)` ?

Reply:
There's a little extra exception handling, to make `@date` `nil` rather than raise a `Date::Error`, I did that to be consistent with all the exception-raising validation logic being in `call`.